reference/profiles.rst

```diff
@@ -131,6 +131,29 @@ They accept patterns too, like ``-s *@myuser/*``, which means that packages that
         [env]
         PATH=[/some/path/to/my/tool]
 
+Tools configurations
+--------------------
+
```

Comment:
Add here a note that the [conf] is an experimental feature

Reply:
Done! 😉